qtglean/include/glean/uuid.h

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef UUID_H
+#define UUID_H
+#include <QObject>
+
+#include "errortype.h"
+
+class UuidMetric final {
+  Q_GADGET
+
+ public:
+  // QML custom types require these three declarations.
+  // See: https://doc.qt.io/qt-6/custom-types.html#creating-a-custom-type
+  UuidMetric() = default;
+  UuidMetric(const UuidMetric&) = default;
```

Comment:
In the tests we need to check if m_id is correctly copied in this copy ctor.

Reply:
See my comments above and below, I do not think this is necessary.